modules/core/private_channel/private_channel.msg

```diff
@@ -74,5 +74,12 @@
   "off": {
     "en_US": "off",
     "de_DE": "aus"
+  },
+  "add_self_permbased": {
+    "en_US": "This char is not an member of the bot. But since your accesslevel is high enough, I added you as one.",
```

Comment:
you should use passive voice, to match the tone and style of the rest of the bot messages

Reply:
so typically, the means that you should not use "I" to have the bot refer to itself.  In this case, I think a message like this would be best: "You have been added as a member." which matches the 'join' message.

Reply:
you may not even need this message at all tho, see my other comment